packages/client/src/pages/contribute/TaggingInterface.tsx

```diff
@@ -53,20 +51,20 @@ const MainView: React.FC<MainViewProps> = (props) => {
 
   return (
     <>
-      {tag ? (
+      {tag && (
         <Box sx={{ justifyContent: 'space-between', display: 'flex', maxWidth: '80%', margin: 'auto' }}>
-          <EntryView
-            entry={tag.entry}
-            width={500}
-            autoPlay={true}
-            pauseFrame="start"
-            mouseOverControls={false}
-            displayControls={true}
-          />
-          <TagForm study={props.study} setTagData={setTagData} />
+          <>
+            <EntryView
+              entry={tag.entry}
+              width={500}
+              autoPlay={true}
+              pauseFrame="start"
+              mouseOverControls={false}
+              displayControls={true}
+            />
+            <TagForm study={props.study} setTagData={setTagData} />
+          </>
         </Box>
-      ) : (
```

Comment:
Don't remove the `NoTagNotification`, when the user has completed all the tags they need something to see